presto-spi/src/main/java/com/facebook/presto/spi/type/TypeSignature.java

```diff
@@ -17,6 +17,7 @@
 import com.fasterxml.jackson.annotation.JsonValue;
 
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
Sure - extracted https://github.com/prestodb/presto/pull/5249.
@martint Can you take a look.